slack/slack.php

```diff
@@ -10,11 +10,13 @@ class SlackPlugin extends Plugin {
   var $config_class = "SlackPluginConfig";
 
   function bootstrap() {
-    Signal::connect('ticket.created', array($this, 'onTicketCreated'), 'Ticket');
+    $config = $this->getConfig();
+    Signal::connect('ticket.created', function ($ticket) use($config) {
```

Comment:
```suggestion
    Signal::connect('ticket.created', function ($ticket) use ($config) {
```

It'd be nice to stay consistent.